spec/lib/tasks/assets_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "assets.rake" do
+  before do
+    AssetManager::Application.load_tasks if Rake::Task.tasks.empty?
+  end
+
+  describe "get_id_by_legacy_url_path" do
+    before do
+      task.reenable # without this, calling `invoke` does nothing after first test
+      allow($stdout).to receive(:puts) # supress Rake task output for cleaner tests
```

Comment:
I don't see any difference with and without this, as you're capturing the output.

Reply:
It suppresses the output in the console (so we don't see lots of rake task output when running `rspec`). But we still want to have tests checking what the output will contain when we run it for real.

Reply:
Have you tried to run it with and without this line? What was the difference?

Reply:
Huh, you're right - it made no difference! I copied it over from https://github.com/alphagov/manuals-publisher/pull/2427 (where it _did_ make a difference) - not sure why it's not needed here, but have now removed it.